### PR TITLE
Added tests for logging metadata

### PR DESCRIPTION
**Pull request by Raalsky**

### CODE REVIEW

File: e2e_tests/integrations/test_huggingface.py [hunk 1]
```diff
@@ -161,3 +167,51 @@ def test_run_access_without_callback_configured(self):
 
         with pytest.raises(Exception):
             NeptuneCallback.get_run(trainer)
+
+    def test_log_parameters_indicator(self, environment):
+        with init_run(project=environment.project, api_token=environment.user_token) as run:
+            base_namespace = "custom/base/path"
```
Comment (shnela) [inline e2e_tests/integrations/test_huggingface.py:173]: What is `base_namespace` for, we have separete test for it in `test_log_with_custom_base_namespace`.

**Review by shnela — CHANGES_REQUESTED** [review]
I don't see check for `eval` namespace, nor `args` and `model.config` log tests.

File: e2e_tests/integrations/test_huggingface.py [hunk 2]
```diff
@@ -161,3 +167,51 @@ def test_run_access_without_callback_configured(self):
...
+            callback = NeptuneCallback(run=run, base_namespace=base_namespace)
+            trainer = Trainer(**self._trainer_default_attributes, callbacks=[callback])
+            trainer.train()
+            run.sync()
+
+            assert run.exists(f"{base_namespace}/trainer_parameters")
+            assert run.exists(f"{base_namespace}/model_parameters")
```
Comment (shnela) [inline e2e_tests/integrations/test_huggingface.py:180]: Don't we want to check if logged values are valid?

File: e2e_tests/integrations/test_huggingface.py (hunk 1 above)
Comment (Raalsky) [reply e2e_tests/integrations/test_huggingface.py:173]: In `test_log_with_custom_base_namespace` we're testing that `log(...)` (a user-exposed) method applies the `base_namespace` as a prefix. In this case we're testing the common parameters (an internal only). It's something really different as the correctness of the preceding doesn't impact the following :wink:

File: e2e_tests/integrations/test_huggingface.py (hunk 2 above)
Comment (Raalsky) [reply e2e_tests/integrations/test_huggingface.py:180]: Ok. Will try

Comment (Raalsky) [reply e2e_tests/integrations/test_huggingface.py:182]: I've added a few simple one that we know they exists + In order to use a structure of `config` I've added proper model config that was introduced here: https://github.com/neptune-ai/neptune-client/pull/954

**Review by shnela — APPROVED** [review]

**Pull request merged**